plugin/trino-clickhouse/src/main/java/io/trino/plugin/clickhouse/ClickHouseClientModule.java

```diff
@@ -33,6 +33,14 @@
 public class ClickHouseClientModule
         implements Module
 {
+    @Provides
```

Comment:
Revert unrelated change.
Ideally we'd want the entire codebase to use formatter but that causes git history difficult to use.